InterfaceJulia/src/TasOneDimesionalRule.jl

```diff
@@ -1,7 +1,8 @@
+# A collection of 1D interpolation functions that return points and
+# weights in column major format.
 module TasOneDimensionalRule
 export clenshaw_curtis
 
-# TODO: Make the output incrementally nested.
```

Comment:
OK, now we can make comments to the Pull-Request. this is good, nothing to change.